regtests/t_cli/src/test_cli.py

```diff
@@ -38,7 +38,7 @@ def get_salt(length=8) -> str:
 
 
 def root_cli(*args):
-    return cli('principal:root;realm:default-realm')(*args)
+    return cli(os.getenv('REGTEST_ROOT_BEARER_TOKEN', 'principal:root;realm:POLARIS'))(*args)
```

Comment:
Question: Does python regression test also use `run.sh` as I didn't see we add the part to fetch token in python scripts?